test/e2e/renderers/checkboxRenderer.spec.js

```diff
@@ -12,6 +12,30 @@ describe('CheckboxRenderer', () => {
     }
   });
 
+  class DataTransferObject {
```

Comment:
Could you move `DataTransferObject` and `getClipboardEvent` to common helper? CopyPaste's tests could use it too.